test

```diff
@@ -158,9 +158,18 @@ function fmt_tests {
 			# ignore revert commits
 			continue
 		fi
-		if ! [[ "$pfx" =~ :$ ]]; then
+
```

Comment:
although that breaks the current merge/revert checking; I think that would then instead be `if [[ "$commitMsg" == Revert* ]]`